src/app/components/lora-selector/lora-selector.component.html

```diff
@@ -79,9 +79,15 @@ <h3 class="text-bold">{{ item.name }}</h3>
 
     <div class="pagination">
       <a [routerLink]="" [class.disabled]="currentPage() <= 1" (click)="currentPage() <= 1 ? null : loadPage(currentPage() - 1, $event)">&lt;</a>
+
       @for (page of pages(); track page) {
-        <a [routerLink]="" (click)="loadPage(page, $event)">{{page}}</a>
+        @if (page === currentPage()) {
+          <a [routerLink]="" class="active">{{page}}</a>
+        } @else {
+          <a [routerLink]="" (click)="loadPage(page, $event)">{{page}}</a>
+        }
```

Comment:
Wouldn't it work better like this?

```suggestion
        <a [routerLink]="" (click)="loadPage(page, $event)" [class.active]="page === currentPage()">{{page}}</a>
```